File: 프로그래머스/1/12916. 문자열 내 p와 y의 개수/문자열 내 p와 y의 개수.js
function solution(s){
    let arr = s.toLowerCase().split('')
    let p = 0; let y=0;
    for(let i=0; i<arr.length; i++){
        if(arr[i] == 'p'){
            p++;
        }else if(arr[i]=='y'){
            y++;
        }
    }
    if( p == y ){
        return true;
    }else{
        return false
    }
}

File: 프로그래머스/1/12916. 문자열 내 p와 y의 개수/문자열 내 p와 y의 개수.py
def solution(s):
    p = 0 
    y = 0
    for curr in s:
        if curr == "p" or curr == "P":
            p += 1
        elif curr == "y" or curr == "Y":
            y += 1
    
    if p == y:
        return True
    else:
        return False